test/e2e/gardener/shoot/create_and_force-delete.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2023 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
```

Comment:
Fix file name (`-` vs `_`)

Reply:
Some other files `upgrade_non-ha_to_node` and `upgrade_non-ha_to_zone` also have the same naming. Will fix all then.